flow/designs/asap7/mock-array/power.tcl

```diff
@@ -22,3 +22,44 @@ report_parasitic_annotation
 report_power
 read_power_activities -scope TOP/MockArray -vcd designs/src/mock-array/MockArrayTestbench.vcd
 report_power
+
+# FIXME add an automated test to check that the top-level power is
+# smaller than the sum of the individual elements.
+#
+# ChatGPT: (4.11 Watts) is slightly greater than the sum of all
+# the individual elements' power (4.0668 Watts).
+# Therefore, the top-level power is not smaller than the sum
+# of all the elements. ​
+
+for {set x 0} {$x < 8} {incr x} {
+  for {set y 0} {$y < 8} {incr y} {
+    puts "Power for ces_${x}_${y}"
+    report_power -instances ces_${x}_${y}
+  }
+}
+
+# FIXME what is the name of the top-level, i.e. how do I do
+# report_power -instances MockArray.
+#
+# Why is there a difference between the output format of
+# "report_power" (top level):
```

Comment:
I suppose there is no need to group instances when you are asking for them to be reported individually.

Reply:
But I want to know the grouping *within* a module. I want to know the combinational power for an Element.

Reply:
I expect that will be a sta enhancement

Reply:
Gotit. Will file a feature request.